### PR TITLE
New ptBR localization

### PR DESCRIPTION
**Pull request by josericardo-fo**

### CODE REVIEW

**Comment by josericardo-fo** [comment]
### IVs
image

### Berries
image

### Egg List
image

### Egg Gatcha
image

**Comment by bennybroseph** [comment]
has some conflicts atm

**Comment by josericardo-fo** [comment]
working

File: src/locales/pt_BR/modifier-type.ts [hunk 1]
```diff
@@ -4,384 +4,406 @@ export const modifierType: ModifierTypeTranslationEntries = {
...
+    "FIRE": "Fogo",
+    "WATER": "Água",
+    "GRASS": "Grama",
+    "ELECTRIC": "Elétrico",
+    "PSYCHIC": "Psíquico",
+    "ICE": "Gelo",
+    "DRAGON": "Dragão",
+    "DARK": "Sombrio",
+    "FAIRY": "Fada",
+    "STELLAR": "Estelar",
```
Comment (bennybroseph) [inline src/locales/pt_BR/modifier-type.ts:407]: tera type is gone now, no?

**Review by bennybroseph — CHANGES_REQUESTED** [review]

File: src/locales/pt_BR/modifier-type.ts (hunk 1 above)
Comment (josericardo-fo) [reply src/locales/pt_BR/modifier-type.ts:407]: oh snap, its true

Comment (josericardo-fo) [reply src/locales/pt_BR/modifier-type.ts]: fixed

**Review by bennybroseph — APPROVED** [review]

**Pull request merged**